app/Child.php

```diff
@@ -40,6 +40,12 @@ class Child extends Model
     ];
 
 
+    /**
+     * These are turned into Date objects on get
+     *
+     * @var array
+     */
+
```

Comment:
remove extra line?

Reply:
Does your editor add these gaps between function doc and function? I find it odd.